### DISCUSSION TITLE
What does the new `Compiler version doesn't include clang or GCC: "c++" "--version"` mean?

### DISCUSSION THREAD

**Discussion by David-Else** [helix-editor/helix#9626] [Q&A] [answered]
I just installed from master and got `Compiler version doesn't include clang or GCC: "c++" "--version"`. Helix runs fine, what should I understand this message means?

EDIT: I am using Debian 12

```
$ gcc --version
gcc (Debian 12.2.0-14) 12.2.0
```

**Comment by pascalkuthe** [comment]
Where does this pop up? On startup? in the log? while building? as a status message?

**Reply by David-Else** [reply]
While building, there was about 4 of the message one after the other when I built the latest master. I am using rustc 1.75.0.

**Reply by pascalkuthe** [reply]
So these are build errors?

**Reply by David-Else** [reply]
Yes, but Helix seems to run fine. I have built from master many times and never seen this error before. I tried to build again, but now there is nothing new in master these errors do not happen again.

EDIT I did update Debian to the latest point version, maybe something changed in GCC?

**Reply by the-mikedavis** [reply, marked as answer]
It's from the newer cc-rs versions, see https://github.com/rust-lang/cc-rs/issues/909

The warning itself seems benign but noisy on debian/ubuntu and derivatives
